apps/comps/app/competitions/[id]/page.tsx

```diff
@@ -16,7 +16,6 @@ import { BreadcrumbNav } from "@/components/breadcrumb-nav";
 import { CountdownClock } from "@/components/clock";
 import { CompetitionInfo } from "@/components/competition-info";
 import CompetitionSkeleton from "@/components/competition-skeleton";
-import { CompetitionVotingBanner } from "@/components/competition-voting-banner";
```

Comment:
@mzkrasner what Linear issue is this PR for? i dont see one that describes removing the banner, but maybe i'm missing something